bats/foreman_helper.bash

```diff
@@ -18,6 +18,23 @@ tForemanVersion() {
   ) | cut -d. -f1-2
 }
 
+tKatelloVersion() {
+  (
+    if tPackageExists tfm-rubygem-katello; then
+      tPackageVersion tfm-rubygem-katello
+    elif tPackageExists rubygem-katello; then
+      tPackageVersion rubygem-katello
+    fi
```

Comment:
any reason not to check the `katello` metapackage instead?

Reply:
The rubygem package was more reliable for the truth, brace yourself -- https://fedorapeople.org/groups/katello/releases/yum/nightly/katello/el7/x86_64/katello-3.18.0-0.4.master.el7.noarch.rpm

Reply:
I wouldn't mind if we eventually get rid of the `katello` metapackage or at least make it fully optional. Because of that, :+1: for looking at the gem. Alternatively, I'd be fine with deriving it from the Foreman version and see if it's at least 2.4 to avoid testing for yet another package version.

Reply:
Bleh @ Katello.rpm

Ack to ewouds idea

Reply:
I thought of trying to use the Foreman version, but given this is a Katello feature, and we can assume lockstep, the past has proven it will not always be true. I've written the code, it works, and I'd prefer to stick with the Katello version here to be as specific as possible.

Reply:
wfm